### DISCUSSION TITLE
Does Deepgram Speech to Text support sentiment analysis ?

### DISCUSSION THREAD

**Discussion by saxena666** [deepgram/community#795] [General help] [answered]
hi, Does Deepgram Speech to Text support sentiment analysis in real time? I understand the Speech to text can be displayed using interim results. Along with this am I able to use this transcription for sentiment analysis in real time too? if yes - how?

also can anyone share any examples, or services built using Deepgram that i can see and experience please?

Thanks in advance

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@sabysaxena We don't currently support Sentiment Analysis on Live or Real time audio. 

For examples of Deepgram being used I'd suggest you checkout these resources:

- https://emilyai.deepgram.com/ - a demo application using speech to text + LLM + text to speech
- https://github.com/deepgram-starters - starter apps in different languages covering pre-recorded, streaming and text to to speech.
- https://github.com/deepgram-devs - opensource projects
